ledger/ledger-api-tests/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuite.scala

```diff
@@ -37,21 +37,28 @@ private[testtool] abstract class LedgerTestSuite {
     )((ec: ExecutionContext) => (_: Seq[ParticipantTestContext]) => testCase(ec))
   }
 
+  /** @param cleanupNewUsers Keeps track of users present before the test case started and after it has finished and deletes difference.
+    *                            NOTE: If set then the test case will not run concurrently.
+    */
   protected final def testGivenAllParticipants(
       shortIdentifier: String,
       description: String,
-      participants: ParticipantAllocation,
+      partyAllocation: PartyAllocation,
       timeoutScale: Double = 1.0,
       runConcurrently: Boolean = true,
       repeated: Int = 1,
       enabled: Features => Boolean = _ => true,
       disabledReason: String = "No reason",
+      cleanupNewUsers: Boolean = false,
   )(
       testCase: ExecutionContext => Seq[ParticipantTestContext] => PartialFunction[
         Participants,
         Future[Unit],
       ]
   ): Unit = {
+    if (cleanupNewUsers) {
```

Comment:
Where else in this method is `cleanupNewUsers` used? Maybe I'm blind but could be worth asking.

Reply:
You're right. It's never used. I missed to delete it alongside some old code that used it.